{
  "messages": [
    {
      "error_id": "",
      "references": [
        {
          "column": 9,
          "file": "/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/SPMDIR-MUST/static/034-MPI-static-get-get-same-array-local-no.c",
          "line": 39
        },
        {
          "column": 9,
          "file": "/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/SPMDIR-MUST/static/034-MPI-static-get-get-same-array-local-no.c",
          "line": 40
        }
      ],
      "text": "Found get operation/call at results-20250730-140308/SPMDIR-MUST/static/034-MPI-static-get-get-same-array-local-no.c:39:9 which is in conflict with get operation/call at results-20250730-140308/SPMDIR-MUST/static/034-MPI-static-get-get-same-array-local-no.c:40:9 before release",
      "type": "Local Data Race"
    }
  ]
}
